src/views/studio/MyVideosView/MyVideosView.tsx

```diff
@@ -135,9 +147,16 @@ export const MyVideosView = () => {
                 onPullupClick={(e) => {
                   e.stopPropagation()
                   handleVideoClick(draft.id, { draft: true, minimized: true })
+                  const index = videoTabs.findIndex((item) => item.id === draft.id)
+                  displaySnackbar({
+                    title: 'Video opened in new tab',
+                    iconType: 'success',
+                    actionText: 'Remove',
+                    onActionClick: () => removeVideoTab(index),
+                  })
```

Comment:
Index here will always be `-1` and this will not work. Try to figure out why, let me know if you struggle

Reply:
this was resolved by returning index from added tab, but it still has to use `removeVideoTab`, which did not operate on current state. I believe it's because of the how current snackbar works. Did a workaround with adding useEffect. Please take a look if that solution is satisfying